balloon/balloon.go

```diff
@@ -74,6 +74,12 @@ type Snapshot struct {
 	Version       uint64
 }
 
+// BulkSnapshot is the struct that has both history and hyper digest and the
+// current version for that rootNode digests.
+type SnapshotBulk struct {
```

Comment:
Why didn't use a type like `type SnapshotBulk []Snapshot`?

Reply:
We want to be homogeneus with the design, like `Events` struct (for example)

Reply:
type snapshotBulk []*Snapshots